docs/source/miniconda.rst

```diff
@@ -32,20 +32,20 @@ which does require administrator permissions.
-   macOS,`Miniconda3 macOS Intel x86 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh>`_,``7406579393427eaf9bc0e094dcd3c66d1e1b93ee9db4e7686d0a72ea5d7c0ce5``
-   ,`Miniconda3 macOS Intel x86 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg>`_,``9195ffba1a6984c81c69649ce976a38455ace5b474c24a4363e5ca65fc72e832``
-   ,`Miniconda3 macOS Apple M1 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.sh>`_,``22eec9b7d3add25ac3f9b60621d8f3d8df3e63d4aa0ae5eb846b558d7ba68333``
-   ,`Miniconda3 macOS Apple M1 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.pkg>`_,``fbb33c5770b10a0d5a0deef746e7499bfaf8ff840d0d517175036dd8449357f6``
-   Linux,`Miniconda3 Linux 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh>`_,``00938c3534750a0e4069499baf8f4e6dc1c2e471c86a59caa0dd03f4a9269db6``
-   ,`Miniconda3 Linux-aarch64 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-aarch64.sh>`_,``48a96df9ff56f7421b6dd7f9f71d548023847ba918c3826059918c08326c2017``
-   ,`Miniconda3 Linux-ppc64le 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-ppc64le.sh>`_,``4c86c3383bb27b44f7059336c3a46c34922df42824577b93eadecefbf7423836``
-   ,`Miniconda3 Linux-s390x 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-s390x.sh>`_,``a150511e7fd19d07b770f278fb5dd2df4bc24a8f55f06d6274774f209a36c766``
+   macOS,`Miniconda3 macOS Intel x86 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh>`_,``bfb81814e16eb450b1dbde7b4ecb9ebc5186834cb4ede5926c699762ca69953b``
+   ,`Miniconda3 macOS Intel x86 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg>`_,``bcc0067864011a93083ff2d6fe7b29e877c1477f24ee9d34b54d0165f8b32f11``
+   ,`Miniconda3 macOS Apple M1 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.sh>`_,``cc5bcf95d5db0f7f454b2d800d52da8b70563f8454d529e7ac2da9725650eb27``
+   ,`Miniconda3 macOS Apple M1 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.pkg>`_,``09d893e44400f61d36daeaa9befff8219a7e0127358d904a4368b2f0ae738df0``
+   Linux,`Miniconda3 Linux 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh>`_,``32d73e1bc33fda089d7cd9ef4c1be542616bd8e437d1f77afeeaf7afdb019787``
+   ,`Miniconda3 Linux-aarch64 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-aarch64.sh>`_,``80d6c306b015e1e3b01ea59dc66c676a81fa30279bc2da1f180a7ef7b2191d6e``
+   ,`Miniconda3 Linux-ppc64le 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-ppc64le.sh>`_,``9ca8077a0af8845fc574a120ef8d68690d7a9862d354a2a4468de5d2196f406c``
+   ,`Miniconda3 Linux-s390x 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-s390x.sh>`_,``0d00a9d34c5fd17d116bf4e7c893b7441a67c7a25416ede90289d87216104a97``
```

Comment:
This PR currently only updates the latest installers... should we also update the other links in that file from 22.11.1 to 23.1.0?

Reply:
Done. Thanks for catching that. I needed to update the conda version in the `create_miniconda_rst.py` script. These scripts need to be improved in general so that updating these pages is not such a headache. I'll create a ticket for this.

Reply:
Created this ticket: https://anaconda.atlassian.net/browse/PKG-1112